CHANGELOG.md

```diff
@@ -5,15 +5,22 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 ## [Unreleased]
 
+### Breaking changes
+
+#### exonum
+
+- `storage::base_index` module becomes private along with `BaseIndex` and
```

Comment:
> becomes

I thought we are not using  Simple Present in chengelog

Reply:
Fixed. :+1: